testing/src/main/java/org/oppia/android/testing/profile/ProfileTestHelper.kt

```diff
@@ -105,6 +105,12 @@ class ProfileTestHelper @Inject constructor(
    */
   fun logIntoNewUser() = logIntoProfile(internalProfileId = 2)
 
+  /**
+   * Login to a new user profile that has no progress for any topics or stories. This relies on other
+   * tests utilizing profile 1 as the default user profile so that profile 2 never has any progress.
+   */
+  fun loginToNewUser() = logIntoProfile(internalProfileId = 2)
```

Comment:
Also, can we have a test in `ProfileTestHelperTest` which test after log into the new user, the profile will not have any progress as mentioned in the comment for `loginToNewUser `.

Reply:
This is not related to my PR. reverted these changes.